src/commands/assets/getAsset.ts

```diff
@@ -53,26 +51,47 @@ export const getAsset = async (
     chalk.dim(`Downloading asset: ${did} with agreement id: ${agreementId}`)
   )
 
-  let results
-  if (argv.password) {
+  const ddo = await nvm.assets.resolve(did)
+  const metadata = ddo.findServiceByType('metadata')
+
+  let isDTP = false
+  if (argv.fileIndex && argv.fileIndex >= 0)
+    isDTP = metadata.attributes.main.files?.[argv.fileIndex].encryption === 'dtp' ? true : false
+  else
+    metadata.attributes.main.files?.forEach( _f => {
+      if (_f.encryption === 'dtp')
+        isDTP = true
+    })
```

Comment:
maybe using `some` works better here

```suggestion
    isDtp = metadata.attributes.main.files?.some( _f => _f.encryption === 'dtp')
    })
```